content/articles/how-dig.markdown

```diff
@@ -43,7 +43,7 @@ The question section displays the DNS question that was sent: "I want A records
 It also gives details about how long the query took, what server was used, when the query was sent, and the size of the DNS packet. 
 
 
-## Dig at a Nameserver
+## Dig at a Name server
```

Comment:
Being in a title, shouldn't "Server" be capitalized as well?

Reply:
actually, 'name' shouldn't be. We don't capitalize h2s in support docs.